src/components/layouts/DashboardLayout/Layout.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { compose, setDisplayName, setPropTypes, defaultProps, mapProps, pure } from 'recompose'
+import { css } from 'aphrodite/no-important'
+import { defaultPropsSpec, propTypes, mapProps as propsMapper } from './lib'
+import tx from '../../../theme/styles'
+import sx from './styles'
+import MainNav from './components/MainNav'
+import DashboardHeader from './components/DashboardHeader/DashboardHeader'
+
+const displayName = 'components/layouts/DashboardLayout'
+
+const enhance = compose(
+  setDisplayName(displayName),
+  setPropTypes(propTypes),
+  defaultProps(defaultPropsSpec),
+  mapProps(propsMapper),
+  pure
+)
+
+function Layout (props) {
+  const {
+    topBarContent,
+    headerContent,
+    mainContent,
+    mainContentContainerProps
+  } = props
+  return (
+    <div className={css(tx.flex, tx.flexDirectionColumn, sx.container)}>
+      <div className={css(tx.textTransformUppercase, tx.fontWeight500, sx.topBarContainer)}>
```

Comment:
i suggest removing `tx.textTransformUppercase` to allow passing any text format client code wants